Makefile

```diff
@@ -74,7 +74,7 @@ start-validator:
 
 .PHONY: install
```

Comment:
You can:
- adapt https://github.com/gardener/gardener-extension-provider-alicloud/blob/8f99926ec07a47105caf6fe25c285d151709842e/Makefile#L23
- adapt https://github.com/gardener/gardener-extension-provider-alicloud/blob/8f99926ec07a47105caf6fe25c285d151709842e/Makefile#L51 and https://github.com/gardener/gardener-extension-provider-alicloud/blob/8f99926ec07a47105caf6fe25c285d151709842e/Makefile#L65